metabase/metabase
# Fix scrolling in filter picker

How to verify:
- Add at least one segment
- Modify `FilterColumnPicker` to have 1000 segments

```
const segmentItems = includeSegments
        ? Array(1000)
            .fill(0)
            .flatMap(() =>
              Lib.availableSegments(query, stageIndex).map(segment => ({
                ...Lib.displayInfo(query, stageIndex, segment),
                segment,
              })),
            )
        : [];
```

- Open the picker
- Ensure scrollbars work as expected

<img width="427" alt="Screenshot 2023-12-05 at 13 41 27" src="https://github.com/metabase/metabase/assets/8542534/00e9fa56-2a1e-4c77-a30c-cf26fff0f5ea">
